test/decorator_test.dart

```diff
@@ -75,16 +72,7 @@ void main() {
     // TODO: Set up saucelabs and run these tests on actual browsers on which we can assert whether
     // our baked-in feature detection is accurate
     group('should identify feature support:', () {
-      void assertFakeRootNode() {
-        if (fakeRootNode == null) {
```

Comment:
fakeRootNode is non-nullable now, so I just removed this

Reply:
Unfortunately I think this changed the behavior of tests. Previously, `fakeRootNode` would be set to a value in the `setUp` of every test (and this `assertFakeRootNode()` verified that that happened), now it's only being set one time, meaning that the same node is being shared across tests. While it apparently works (tests pass), I think it would be better to use a new node for each test like we were doing.

Reply:
Looking at the code again I don't think we're changing the logic of how or when `fakeRootNode` is set to something. The only place it is assigned to a new DivElement is in `sharedSetup` which is called from each groups setUp method. The 2nd group also has a setup method, but it wasn't at the top and is easy to miss, so I moved it to the top. So I think it is using a new element each time.

Reply:
Yup this was my bad, I was thinking of it as `late final` and so I was thinking it would only ever be set the first time.. but that's not the case and it still gets set in each setUp. Sorry about the confusion!

Reply:
No thank you! This would have been something I could have easily overlooked so it was good for me to give it a closer look.